pom.xml

```diff
@@ -35,6 +35,11 @@
          <scope>test</scope>
       </dependency>
       <dependency>
+        <groupId>org.apache.commons</groupId>
+        <artifactId>commons-configuration2</artifactId>
```

Comment:
I must have missed your response, is using HashCodeBuilder necessary? if not, remove this import -- its not needed